test/e2e/net_policy_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 The etcd-operator Authors
```

Comment:
Here that would be `Copyright 2018 The sensu-operator Authors`

Reply:
done